README.md

```diff
@@ -41,7 +41,13 @@ root@raspberrypi:~#
 
 ##### Prerequisites
 
-[KVM](https://help.ubuntu.com/community/KVM/Installation)
+[KVM](https://help.ubuntu.com/community/KVM/Installation) and rdiff
+
+
+(For Ubuntu 16.04)
+```
+# apt-get install qemu-kvm libvirt-bin ubuntu-vm-builder bridge-utils rdiff
```

Comment:
Can you please confirm that above instructions have been tested on a fresh install using [pre-built binaries](https://github.com/myechuri/ukd/tree/master/releases/download/v0.1dev)? i'll merge the PR once you confirm. Thanks a lot for your contribution!